hudi-spark-datasource/hudi-spark/src/test/scala/org/apache/hudi/functional/TestCOWDataSourceStorage.scala

```diff
@@ -185,7 +188,7 @@ class TestCOWDataSourceStorage extends SparkClientFunctionalTestHarness {
     val hoodieIncViewDF1 = spark.read.format("org.apache.hudi")
       .option(DataSourceReadOptions.QUERY_TYPE.key, DataSourceReadOptions.QUERY_TYPE_INCREMENTAL_OPT_VAL)
       .option(DataSourceReadOptions.BEGIN_INSTANTTIME.key, "000")
-      .option(DataSourceReadOptions.END_INSTANTTIME.key, firstCommit)
+      .option(DataSourceReadOptions.END_INSTANTTIME.key, completionTime1)
```

Comment:
Should this be from `completionTime1` to `completionTime2`?

Reply:
this query is to pull the first commit. so the start time should be a completion time that's before the 1st commit's completion time. And the end time should be completionTime1. 

Basically to fetch commit in this range: `(completionTime1, completionTime1]`

Reply:
I'll add a new start completion time in this test to avoid confusion: 
```
val beforeCompletionTime1 = HoodieInstantTimeGenerator.instantTimeMinusMillis(completionTime1, 1)
```

Reply:
Sg